test/honeypot/honeypot-pipeline.test.ts

```diff
@@ -0,0 +1,221 @@
+import { expect as expectCDK, haveResource, haveResourceLike } from '@aws-cdk/assert'
+import * as cdk from '@aws-cdk/core'
+import { HoneypotPipelineStack } from '../../src/honeypot/honeypot-pipeline'
+import { getContextByNamespace } from '../../src/context-helpers'
+import { FoundationStack } from '../../src/foundation-stack'
+import helpers = require('../../test/helpers')
+
+describe('CodeBuild actions', () => {
+  beforeEach(() => {
+    helpers.mockDockerCredentials()
+  })
+  const stack = () => {
+    process.env.CDK_CONTEXT_JSON = JSON.stringify({ dockerhubCredentialsPath: '/path/to/oauth' })
+    const app = new cdk.App()
+    // WHEN
+    const env = {
+      name: 'test',
+      domainName: 'test.edu',
+      domainStackName: 'test-edu-domain',
+      networkStackName: 'test-network',
+      region: 'test-region',
+      account: 'test-account',
+      createDns: true,
+      useVpcId: '123456',
+      slackNotifyStackName: 'slack-test',
+      createGithubWebhooks: false,
+      useExistingDnsZone: false,
+      notificationReceivers: 'test@test.edu',
+      alarmsEmail: 'test@test.edu',
+    }
+    const hostnamePrefix = 'honeypot-test'
+    const honeypotContext = getContextByNamespace('hneypot')
```

Comment:
```suggestion
    const honeypotContext = getContextByNamespace('honeypot')
```

Not sure if this typo impacts your tests - if you fix, be sure to re-run